sqldb/sqlc/migrations/000001_invoices.up.sql

```diff
@@ -78,15 +102,15 @@ CREATE INDEX IF NOT EXISTS invoice_feature_invoice_id_idx ON invoice_features(in
 CREATE TABLE IF NOT EXISTS invoice_htlcs (
     -- The id for this htlc. Used in foreign keys instead of the 
     -- htlc_id/chan_id combination.
-    id INTEGER PRIMARY KEY,
+    id BIGINT PRIMARY KEY,
+    
+    -- Short chan id indicating the htlc's origin. uint64 stored as text.
+    chan_id TEXT NOT NULL, 
```

Comment:
In my experience composite primary keys can be a bit tricky at times (joins get more complex, etc), so I'm not sure we should go in that direction unless there's a compelling feature or workflow that would allow. As is any table ref'ing can just use the main ID here, and handle any conflicts on that level vs needing to handle both the htlc and chan id.

Reply:
Have to agree here with @Roasbeef. Composite keys sound very good on paper, but working with them can be difficult. Having just an extra ID as a point of reference is as simple as it can get.